src/Resources/config/services.yml

```diff
@@ -14,6 +14,7 @@
 
 services:
     surfnet_yubikey_api_client.verification_service:
+        public: true
```

Comment:
This smells ;) 

Can you make it private and force use via DI? This is a good point in time to make things more strict!

Reply:
@MKodde it is because otf this test
`src/Tests/DependencyInjection/SurfnetYubikeyApiClientExtensionTest.php`

Reply:
Thanks for the clarification! And I have this nitpick response:

Can we add the `public: true` override in a services_test.yaml file? This construct might not be clear (enough) at this point. 

It's not that important, so feel free to put aside my feedback.